src/main/kotlin/no/nav/klage/dokument/service/DokumentUnderArbeidService.kt

```diff
@@ -937,7 +937,7 @@ class DokumentUnderArbeidService(
                     documents = vedlegg.map {
                         DocumentsChangedEvent.DocumentChanged(
                             id = it.id.toString(),
-                            parentId = if (it is DokumentUnderArbeidAsVedlegg) it.parentId.toString() else null,
+                            parentId = it.parentId.toString(),
```

Comment:
Greit den ble fjernet :) Hadde beholdt den med vilje, men når jeg tenker på det så er det ikke nødvendig for det jeg tenkte da.